docs/Architecture.md

```
@@ -0,0 +1,90 @@
+| Core | `core:*` | 네트워크, 디자인 시스템, 알람, 미디어 등 인프라성 로직                                     | 필요한 모듈에 `implementation`으로 의존 |
+
+### 인터페이스 우선 접근
+도메인 계약(`domain/src/main/java/com/yapp/domain/repository/FortuneRepository.kt`)은 어떠한 프레임워크에도 의존하지 않습니다.
+
+```kotlin
+interface FortuneRepository {
+    val fortuneCreateStatusFlow: Flow<FortuneCreateStatus>
+    suspend fun getFortune(fortuneId: Long): Result<Fortune>
+    suspend fun markFortuneSeen()
+    // ...
+}
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)